libs/authorizenet/src/drivers/transformers/authorize-net-transformer.service.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable } from '@angular/core';
+
+import { DaffPaymentTransformerService } from '@daffodil/checkout';
+
+import { AuthorizeNetRequest } from '../../models/request/authorize-net-request';
+import { AuthorizeNetResponse } from '../../models/response/authorize-net-response';
+import { DaffAuthorizeNetTokenRequest } from '../../models/request/authorize-net-token-request';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class DaffAuthorizeNetTransformerService implements DaffPaymentTransformerService<DaffAuthorizeNetTokenRequest> {
+	transformOut(request: DaffAuthorizeNetTokenRequest): AuthorizeNetRequest {
+		return {
+			cardData: {
+				cardNumber: request.creditCard.cardnumber,
+				cardCode: request.creditCard.securitycode,
+				month: request.creditCard.month,
+				year: request.creditCard.year
+			},
+			authData: {
+				clientKey: request.authData.clientKey,
```

Comment:
This should probably come from a configuration object and be a part of the driver call, not the transformer.

Reply:
I'm going to add a config object, but all the work around that is in the paypal PR, so can we push this PR without this change, then push the paypal, then I'll come back to this?

Reply:
I decided to separate this module completely from checkout payment. So I added the configuration token